ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -372,102 +372,98 @@ export const ApplicationsList = (props: RouteComponentProps<{}>) => {
                     {ctx => (
                         <ViewPref>
                             {pref => (
-                                <Page
-                                    key={pref.view}
-                                    title={getPageTitle(pref.view)}
-                                    useTitleOnly={true}
-                                    toolbar={{breadcrumbs: [{title: 'Applications', path: '/applications'}]}}
-                                    hideAuth={true}>
```

Comment:
Did you remove `hideAuth={true}>` on purpose for the Page on line 388?

Reply:
Right, I've refactored the Page toolbar such that we don't need special handling for the `ApplicationsList` page anymore. Previously, the `ApplicationsList` had `useTitleOnly` set to true because it rendered its own bespoke top bar / tool bar. After the refactor, it's no longer bespoke.